### DISCUSSION TITLE
Old AMD GPU users?

### DISCUSSION THREAD

**Discussion by RenanOSX** [YellowRoseCx/koboldcpp-rocm#49] [General]
So I have an old gpu (RX580 8GB) and I think that there are more people like me with old gpus, if there was enough people maybe we could get support for older gpus?

**Comment by nielsijzerman** [comment]
You apparently can run Ubuntu with an older rocm, haven't tried it yet, might do so tonight. AMD dropped support in rocm 4.0. And I have tried a run with Vulkan, not tested if its faster or weirder then openblas. I run it witj Voxta, it has a benchmark. Not nearly as fast as my 4070, ok for chat and adventure mode I think. TTS is also on the testing list. Long night.

https://github.com/Grench6/RX580-rocM-tensorflow-ubuntu20.4-guide
